static/app/components/group/externalIssuesList/streamlinedExternalIssueList.tsx

```diff
@@ -78,8 +79,9 @@ export function StreamlinedExternalIssueList({
               </Tooltip>
             </ErrorBoundary>
           ))}
-          {integrations.length
-            ? integrations.map(integration => {
+          {integrations.length ? (
+            <ScrollCarousel>
```

Comment:
Are you sure you want to use the carousel for this? IMO the buttons seem better wrapping to multiple lines